include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -365,48 +386,46 @@ struct __brick_copy_n<_ExecutionPolicy, oneapi::dpl::__internal::__enable_if_het
     }
 };
 
-template <typename _ExecutionPolicy>
-struct __brick_copy<_ExecutionPolicy, oneapi::dpl::__internal::__enable_if_hetero_execution_policy<_ExecutionPolicy>>
+template <typename _BackendTag, typename _ExecutionPolicy>
+struct __brick_copy<__hetero_tag<_BackendTag>, _ExecutionPolicy, void>
```

Comment:
Is this `void` at the very end because we now have typename = void in a primary template?

Reply:
Fixed